wave_vision/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ WAVE_ADD_MODULE(${PROJECT_NAME}
     Eigen3::Eigen
     Boost::filesystem
     opencv_core opencv_features2d opencv_calib3d opencv_imgproc opencv_highgui
+    opencv_videoio opencv_xfeatures2d
```

Comment:
I'm not using xfeatures. Anything from the contrib modules I don't think we can use since libwave is open source...our licensing would have to change.

Reply:
If you could just remove it for now that would be great :)

Reply:
WAIT I just read the message. What depends on xfeatures?!